openg2p/models/beneficiary_to_batch_wizard.py

```diff
@@ -75,7 +75,7 @@ def create_batch(self):
                     {
                         "name": self.batch_name
                         + "-"
-                        + str(datetime.now().strftime("%d%m%y-%I:%M")),
+                        + str(datetime.now().strftime("d-%m-%Y-%H:%M")),
```

Comment:
Wasn't this date time change made in the previous PR?

Reply:
error in date format
d -> %d 
will be pushed in next commit